### ISSUE TITLE
Forms - Indicate Required Content with an Asterisk

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#810]
Instead of the current "case/method/org needs a title" text at the bottom of forms, use a red asterisk beside the required field:

![required content asterisk](https://user-images.githubusercontent.com/13560530/40325179-ae86c9f2-5cef-11e8-828c-d6bbbddc9df2.png)

[Design Files](https://drive.google.com/open?id=1IXeKcHBXyrK05o4Y08xjbJeAsZltRLRo)

If a user tries to publish content without filling out the required field, could use the same style as option 6 in https://github.com/participedia/frontend/issues/593#issuecomment-390384086 with "this is a required field" as prompt text. OR just jump to the field with the asterisk. 

REF: https://github.com/participedia/frontend/issues/808

**Comment by scottofletcher** [comment]
I think we decided the mandatory fields in quick submit are:
Title
Brief Description
Links

@plscully does that sound about right? Those are the things I ask for when someone is entering a stub on .net

**Comment by plscully** [comment]
Perhaps we want to require only title and brief description.  For ex., an impromptu street action is unlikely to have a link

**Comment by scottofletcher** [comment]
I still think we should require a link - it can also be to a page about the issue the action was taken in response to (I think the instructional text makes that clear but we can always change it). Without this being a required field, people tend not to add any sources to stubs which disincentives other users filing them out

**Comment by plscully** [comment]
@scottofletcher I see your point. At the same time, it looks like we are seeking a balance between on the one hand, avoiding barriers that will prevent people from using Quick Submit, and on the other hand collecting links that prompt other users to build on the initial entry.

**Comment by scottofletcher** [comment]
@plscully point taken - I think maybe this could be another task for the 'rapid response' RA - they should 1) monitor the interwebs for topical/current PPedia-relevant initiatives/material, and 2) monitor PPedia for new entries and immediately fix them up by adding links, editing prompts, etc.

**Comment by jesicarson** [comment]
We decided only title is required.

**Comment by plscully** [comment]
Ok - Let's give it try and see how users respond

**Comment by scottofletcher** [comment]
Wait, who's 'we' deciding only on title?? Brief description was _definitely_ something to be mandatory

**Comment by jesicarson** [comment]
I totally see where you might think it should be mandatory, but I strongly suggest that that's not ideal. You don't want mandatory things as a barrier for entry. Not a good idea from a usability perspective. You need minimum one thing (a title) to distinguish a new case.  (Side note: We decided to make the short descrip tweet length for social sharing, and for the list view on the homepage to display something useful, but never intended it to be a mandatory field). 
Think about it this way... If you're working in the field or at a conference or out for coffee and want to quick submit an idea because you just had a convo with someone and they told you about like "Theatre of the Oppressed" or something, but you have to run to the next thing and dont want to forget - what do you do? Quick subbbb! But if then the site is like, oh wait you have to write a description too, and your publish fails.... what do you do? Give up and leave the site and probably never come back. Sad face.
Gotta find the balance between students and ra's who are paid to do entries, and actually being able to say our site is a useful tool for people who don't have as much time to dedicate but still find it interesting enough to contribute to - a NEW audience that we hope xyz can cater to. We haven't had a chance to test that yet which is why we are pushing so hard to launch early and often...

**Comment by plscully** [comment]
I agree with Jesi -- let's give it a try. @scottofletcher -- I can see how you might imagine this creating a nightmare scenario where we end up with lots of stubs that are hard to make sense of. If that ends up happening, we will find a way to deal with it. I won't be expecting you or the other editors to clean up every stub that's submitted.

**Comment by scottofletcher** [comment]
Ok, I concede! I think as a compromise I'd like to have an admin/editor tool that allows us to classify and sort by level of completeness. I'm going to need a way to quickly get to all the entries that only have a title so I can update them w/a brief description and a link. Yes, it's important that barriers to use are low for authors but what about readers? People are going to get frustrated and not use PPedia as a source if all their searches return entries with a title and nothing else. 
(Pat, that'll be a task for the response team - their daily workflow can be: 1) check newly submitted stubs and add at least a brief description, link, and some data, 2) search the interwebs for new/updated cases/methods/orgs and create PPedia entries)

**Comment by scottofletcher** [comment]
@plscully yes, that's imperative!
